<?xml version="1.0"?>

<launch>
  <!-- This launches a mongo db server -->
  <include file="$(find flor_walk_monitor)/launch/mongodb.launch" />

  <!-- This launches the walk monitor node -->
  <node name="flor_walk_monitor" pkg="flor_walk_monitor" type="walk_monitor_node" output="screen" respawn="true">
    <rosparam file="$(find flor_walk_monitor)/config/walk_monitor_params.yaml" command="load" />
    <rosparam file="$(find flor_footstep_planner)/config/atlas_params.yaml" command="load" />
    <param name="log_folder" value="$(find flor_walk_monitor)/results/" />
  </node>
</launch>
